packages/xchain-client/src/types.ts

```diff
@@ -59,6 +60,17 @@ export type TxParams = {
   memo?: string // optional memo to pass
 }
 
+export type TxOfflineParams = {
+  walletIndex?: number // send from this HD index
+  asset?: Asset
+  amount: BaseAmount
+  recipient: Address
+  memo?: string // optional memo to pass
+  from_balance: string
+  from_account_number: string
+  from_sequence: string
+}
```

Comment:
Keep it DRY
```ts
export type TxOfflineParams = TxParams & {
  from_balance: BaseAmount
  from_account_number: string
  from_sequence: string
}
```
Note change of  `from_balance: string` -> `from_balance: BaseAmount`